pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>org.carlspring</groupId>
     <artifactId>properties-injector</artifactId>
-    <version>1.8-SNAPSHOT</version>
+    <version>1.9-SNAPSHOT</version>
```

Comment:
(As described above, we still haven're release `1.8`, so there's no need to bump the version here yet).